src/index.ts

```diff
@@ -1,3 +1,5 @@
+// eslint-disable-next-line import/no-unresolved
+import './locales/index.js'
```

Comment:
sorry for missing this on first review 🤦‍♂️

Reply:
I am the one who missed it!